src/nomad_material_processing/__init__.py

```diff
@@ -80,11 +82,11 @@ class Parallelepiped(Geometry):
     m_def = Section()
     height = Quantity(
         type=float,
-        description='The z dimension of the parallelepiped.',
+        description='The z dimension (or thickness) of the parallelepiped.',
```

Comment:
Does this mean that this is always the smallest dimension? If we have a thin cubic rod for example, the height could be the largest and if you asked people about the thickness of the rod they would probably give you the x or the y. I guess the same can be said for the length though...